### PR TITLE
Admin edit word and answers

### PR DESCRIPTION
**Pull request by trungnd95**

### CODE REVIEW

File: app/controllers/admin/words_controller.rb [hunk 1]
```diff
@@ -36,6 +36,28 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    respond_to do |format|
+      if @word.update_attributes word_params
+        byebug
```
Comment (tiennguyenuet) [inline app/controllers/admin/words_controller.rb]: delete byebug

File: app/controllers/admin/words_controller.rb [hunk 2]
```diff
@@ -36,6 +36,28 @@ def create
...
+        format.html do
+          flash[:success] = "page.admin.words.edit.success"
```
Comment (tiennguyenuet) [inline app/controllers/admin/words_controller.rb]: I18n

File: app/controllers/admin/words_controller.rb [hunk 3]
```diff
@@ -36,6 +36,28 @@ def create
...
+          redirect_to admin_words_path
+        end
+        format.json{render @word, status: :ok}
+      else
+        format.html do
+          flash[:danger] = "page.admin.words.edit.danger"
```
Comment (tiennguyenuet) [inline app/controllers/admin/words_controller.rb]: I18n

**Comment by dieunb** [comment]
OK

**Pull request merged**